shared/src/version/mod.rs

```diff
@@ -57,6 +57,8 @@ impl NetworkVersion {
     pub const V20: Self = Self(20);
     /// Watermelon (builtin-actors v12)
     pub const V21: Self = Self(21);
+    /// TBD builtin-actors v123
```

Comment:
Small nit on the `builtin-actors v123` here, it should `v13`

Reply:
I have fat fingers this morning!! 🤕